app/react/Attachments/components/Attachment.js

```diff
@@ -131,7 +131,7 @@ export class Attachment extends Component {
   }
 
   handleClickOutside(e) {
-    if (!this.myRef.current.contains(e.target)) {
+    if (this.myRef.current && !this.myRef.current.contains(e.target)) {
```

Comment:
Please can you additionally change the name of the reference variable `myRef` to something more specific to their use like `attachmentActionsRef`?